src/app/content/components/ContentLink.tsx

```diff
@@ -17,19 +17,23 @@ interface Props extends React.HTMLProps<HTMLAnchorElement> {
     shortId: string;
     title: string;
   };
+  onClick?: () => void;
   navigate: typeof push;
   currentPath: string;
   className?: string;
 }
 
 // tslint:disable-next-line:variable-name
-export const ContentLink: SFC<Props> = ({book, page, currentPath, navigate, ...props}) => {
+export const ContentLink: SFC<Props> = ({book, page, currentPath, navigate, onClick, ...props}) => {
   const {url, params} = getBookPageUrlAndParams(book, page);
   const relativeUrl = toRelativeUrl(currentPath, url);
 
   return <a
     onClick={(e) => {
       e.preventDefault();
+      if (onClick) {
```

Comment:
Other code uses `onClick && onClick()` syntax; is there a reason for doing it differently here? Or is that just selective-memory?

Reply:
the linter actually complains about doing that in a block. it fails the `no-unused-expression` rule. i think we might do that outside of blocks in `() => thing && thing()` expressions